server/config.ts

```diff
@@ -96,6 +96,6 @@ export default {
   domain: get('INGRESS_URL', 'http://localhost:3000', requiredInProduction),
   dpsHomeUrl: get('DPS_URL', 'http://localhost:3000/', requiredInProduction),
   featureToggles: {
-    editGoalsEnabled: Boolean(get('EDIT_GOALS_ENABLED', false)),
+    editGoalsEnabled: Boolean(get('EDIT_GOALS_ENABLED', true)),
```

Comment:
Any reason why you changed this to `true` ?
I'd be tempted to leave it as `false`, so that the only way you can enable the feature is by setting the env var `EDIT_GOALS_ENABLED` to `true` in your `.env` file or helm deployment?

Unless theres some subtlety here that I've missed?

Reply:
Oops that was me accidentally checking in my lazy way of enabling the feature toggle locally. Will use `.env` in future!